tests/components/aidot/__init__.py

```diff
@@ -0,0 +1,173 @@
+"""Tests for the aidot integration."""
+
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.aidot import (
+    async_setup_entry,
+    async_unload_entry,
+    cleanup_device_registry,
+)
+from homeassistant.components.aidot.const import DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+
+@pytest.fixture
+def mock_hass():
+    """Fixture to create a mock HomeAssistant instance."""
+    hass = AsyncMock(spec=HomeAssistant)
```

Comment:
There's already a `hass` fixture that should be used. Please see existing tests for how to do things. Don't invent your own patterns.